tensorflow/compiler/xla/service/cpu/build_defs.bzl

```diff
@@ -10,3 +10,14 @@ def runtime_copts():
       "//tensorflow:android": ["-O2"],
       "//conditions:default": []
   }))
+
+def append_polly_dep_if_enabled(basic_deps):
```

Comment:
Might be better to use the same pattern as in `runtime_copts` -- return `["@llvm//:polly"]` or not, and then append the result of `polly_deps()` explicitly in the calling rule.